src/main/osd/osd.c

```diff
@@ -545,6 +548,19 @@ void osdInit(displayPort_t *osdDisplayPortToUse, osdDisplayPortDevice_e displayP
     }
 }
 
+#ifdef USE_GPS_LAP_TIMER
+void printLapTime(char* buffer, uint32_t time) {
+    if (time != 0) {
+        timeRound = time + 5;
+        const int timeSeconds = timeRound / 1000;
+        const int timeDecimals = (timeRound % 1000) / 10;
```

Comment:
`time` with the postfix "Ms" gives a hint about what time format goes into the function. Also, when calling the function, a copy of the time input is made, so you can directly manipulate `timeMs`. Therefore you don't need the variable `timeRound`.
```suggestion
void printLapTime(char *buffer, uint32_t timeMs) {
    if (timeMs != 0) {
        timeMs += 5;
        const int timeSeconds = timeMs / 1000;
        const int timeDecimals = (timeMs % 1000) / 10;
```